starky/src/cross_table_lookup.rs

```diff
@@ -327,12 +331,17 @@ pub(crate) fn cross_table_lookup_data<'a, F: RichField, const D: usize, const N:
     constraint_degree: usize,
 ) -> [CtlData<'a, F>; N] {
     let mut ctl_data_per_table = [0; N].map(|_| CtlData::default());
-    for CrossTableLookup {
-        looking_tables,
-        looked_table,
-    } in cross_table_lookups
-    {
-        log::debug!("Processing CTL for {:?}", looked_table.table);
+    for CrossTableLookup { looking_tables } in cross_table_lookups {
+        log::debug!(
+            "Processing CTL for {:?}",
+            looking_tables
+                .iter()
+                .map(|table| table.table)
+                .counts()
+                .into_iter()
+                .sorted()
+                .collect::<Vec<_>>()
+        );
```

Comment:
At this point I think I may prefer just specifying for which table the CTL is, i.e. 

```suggestion
        log::debug!(
            "Processing CTL for STARK table {:?}",
            looking_tables
                .last().
                .expect("We always have a looked table.")
                .table
        );
```

or otherwise removing it altogether. The current display doesn't carry much meaning (especially with the sorting which kind of hides which table is being looked at. For example this is our zkEVM output with this implem:

```bash
[DEBUG starky::cross_table_lookup] Processing CTL for [(0, 1), (2, 1)]
[DEBUG starky::cross_table_lookup] Processing CTL for [(1, 1), (2, 4)]
[DEBUG starky::cross_table_lookup] Processing CTL for [(2, 1), (4, 1)]
[DEBUG starky::cross_table_lookup] Processing CTL for [(3, 1), (4, 1)]
[DEBUG starky::cross_table_lookup] Processing CTL for [(3, 1), (4, 1)]
[DEBUG starky::cross_table_lookup] Processing CTL for [(2, 1), (4, 5), (5, 1)]
[DEBUG starky::cross_table_lookup] Processing CTL for [(1, 32), (2, 7), (4, 136), (6, 1)]
```